apps/server/src/shared/infra/oauth-provider/hydra/hydra.service.ts

```diff
@@ -47,6 +80,18 @@ export class HydraService extends OauthProviderService {
 		return response;
 	}
 
+	rejectConsentRequest(challenge: string, body: RejectRequestBody): Promise<ProviderRedirectResponse> {
+		return this.put<ProviderRedirectResponse>('consent', 'reject', challenge, body);
+	}
+
+	rejectLoginRequest(challenge: string, body: RejectRequestBody): Promise<ProviderRedirectResponse> {
+		return this.request<ProviderRedirectResponse>(
```

Comment:
refactor to use generalized method put<T>

Reply:
done